lib/observability/prometheus.yml

```
@@ -0,0 +1 @@
+ 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)